package.json

```
@@ -81,7 +81,7 @@
     "prepublishOnly": "yarn build",
     "build": "tsc",
     "watch": "tsc -w",
-    "lint": "eslint {src,test,factory}/**/*.ts",
+    "lint": "eslint {src,factory}/**/*.ts test/*.ts test/**/*.ts",
```

This should actually be

```suggestion
    "lint": "eslint '{src,test,factory}/**/*.ts'",
```